peachjam/docx.py

```diff
@@ -6,6 +6,9 @@
 
 logger = logging.getLogger(__name__)
 
+# subprocess timeout (in seconds)
+TIMEOUT = 60 * 30
+
 
 def convert_docx_to_pdf(docx_file):

```

Comment:
This can also be adjusted to call https://github.com/laws-africa/docpipe/blob/main/docpipe/soffice.py#L76 so that you don't need to duplicate the timeout.